Error: Cannot find module '.../node_modules/node-sass/node_modules/pangyp/bin/node-gyp' during `maji new` on OS X 10.11

```
$ node -v
v5.0.0
$ npm -v
3.3.9
$ maji new org.example.my-awesome-app my-app/
```

During the process, the following error is shown:
```
> node-sass@2.0.1 postinstall /Users/patrickbaselier/code/sandbox/my-app/node_modules/node-sass
> node scripts/build.js

module.js:339
    throw err;
    ^

Error: Cannot find module '/Users/patrickbaselier/code/sandbox/my-app/node_modules/node-sass/node_modules/pangyp/bin/node-gyp'
    at Function.Module._resolveFilename (module.js:337:15)
    at Function.Module._load (module.js:287:25)
    at Function.Module.runMain (module.js:457:10)
    at startup (node.js:136:18)
    at node.js:972:3
Build failed
```

`maji new` seems to finish without errors, though.

--- a/package.json
+++ b/package.json
@@ -2,27 +2,30 @@
   "name": "maji",
   "version": "1.0.0",
   "main": "lib/index.js",
-  "bin" : { "maji" : "./lib/cli.js" },
+  "bin": {
+    "maji": "./lib/cli.js"
+  },
   "repository": {
     "type": "git",
     "url": "https://github.com/kabisaict/maji"
   },
   "devDependencies": {
-    "coffee-script": "^1.9.0",
     "backbone": "~1.1.2",
     "backbone.marionette": "^1.8.1",
     "backbone.wreqr": "^1.3.1",
+    "coffee-script": "^1.9.0",
+    "coffeeify": "^1.0.0",
     "jquery": "^2.1.3",
-    "underscore": "^1.6.0",
     "karma": "^0.12.31",
+    "karma-browserify": "^4.2.1",
+    "karma-chai-plugins": "^0.1.3",
     "karma-mocha": "^0.1.10",
     "karma-mocha-reporter": "^0.3.1",
-    "karma-chai-plugins": "^0.1.3",
-    "sinon-as-promised": "^4.0.0",
     "karma-phantomjs-launcher": "^0.1.4",
-    "karma-browserify": "^4.2.1",
     "karma-sinon-chai": "^0.3.0",
-    "coffeeify": "^1.0.0"
+    "sinon-as-promised": "^4.0.0",
+    "underscore": "^1.6.0",
+    "mocha": "^2.3.4"
   },
   "dependencies": {
     "i18n-js": "http://github.com/fnando/i18n-js/archive/v3.0.0.rc8.tar.gz",

--- a/project_template/package.json
+++ b/project_template/package.json
@@ -38,11 +38,11 @@
     "karma-mocha-reporter": "~0.3.1",
     "karma-sinon-chai": "~0.3.0",
     "karma-phantomjs-launcher": "~0.1.4",
-    "node-sass": "~2.0.1",
+    "node-sass": "~3.4.2",
     "onchange": "~1.0.0",
     "tiny-lr": "~0.1.5",
     "uglifyify": "~3.0.1",
-    "vinyl-fs": "~0.3.13",
+    "vinyl-fs": "~2.2.1",
     "watchify": "~2.6.0",
     "yamlify": "~0.1.2"
   },

